src/dockerfile-add-scanner/cli/scan.go

```diff
@@ -388,13 +388,19 @@ func extractURLToPath(u *url.URL, path string, decompress decompress) error {
 			return err
 		}
 
+		target := filepath.Join(path, header.Name)
+
+		if strings.Contains(target, "..") {
```

Comment:
LGTM.

If path ever contains .. (such as /usr/../tmp/) this would fail and it would make more sense to check just header.Name, but currently path is always from TmpDir it seems so it is ok.

Reply:
That is a good point. I could move it to check on `header.Name`, which would be cleaner. I don't think `path` ever would contain that, but we should be precise. I will redo this part.

Reply:
Changed and pushed out. Needs another approval, though.